resources/js/features/games/components/TopAchieversMainRoot/TopAchieversMainRoot.tsx

```diff
@@ -0,0 +1,52 @@
+import { router } from '@inertiajs/react';
+import type { FC } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import { FullPaginator } from '@/common/components/FullPaginator';
+import { GameBreadcrumbs } from '@/common/components/GameBreadcrumbs';
+import { GameHeading } from '@/common/components/GameHeading/GameHeading';
+import { usePageProps } from '@/common/hooks/usePageProps';
+
+import { TopAchieversList } from './TopAchieversList';
+
+export const TopAchieversMainRoot: FC = () => {
+  const { game, paginatedUsers } = usePageProps<App.Platform.Data.GameTopAchieversPageProps>();
+
+  const { t } = useTranslation();
+
+  const handlePageSelectValueChange = (newPageValue: number) => {
+    router.visit(
+      route('game.top-achievers.index', {
+        game: game.id,
+        _query: { page: newPageValue },
+      }),
+    );
+  };
+
+  return (
+    <div>
+      <GameBreadcrumbs
+        game={game}
+        system={game.system}
+        t_currentPageLabel={t('Game Top Achievers')}
+      />
+      <GameHeading game={game}>{t('Game Top Achievers')}</GameHeading>
```

Comment:
minor: I'm wondering if "Top Achievers" as a label may make more sense than "Game Top Achievers".

Given these labels are part of either the game breadcrumbs or the game heading, "Game" in the label could perhaps be redundant.

Reply:
Agreed. Changed.